content/kr/docs/tutorials/_index.md

```diff
@@ -7,65 +7,65 @@ content_template: templates/concept
 
 {{% capture overview %}}
 
-이번 섹션에는 Kubernetes 설명서 자습서가 포함되어 있습니다. 튜토리얼에서는 보다 보다 큰 방향을 갖고 접근합니다. 튜토리얼에는 여러 섹션이 있으며, 섹션별로 진행할 단계가 있습니다. 각 자습서를 살펴보기 전에 나중에 참조 할 수 있도록 표준화 용어집 페이지도 살펴보시기 바랍니다. 
+이번 섹션에는 Kubernetes 튜토리얼이 포함되어 있습니다. 튜토리얼에서는 개별 [작업](/docs/tasks/)에 국한되지 않고 보다 넓은 시야에서 접근합니다. 튜토리얼에는 여러 섹션이 있으며, 섹션별로 진행할 단계가 있습니다. 각 튜토리얼를 살펴보기 전에 나중에 참조할 수 있도록 [표준 용어집 페이지](/docs/reference/glossary/)도 살펴보시기 바랍니다. 
 
 {{% /capture %}}
 
 {{% capture body %}}
 
 ## 기본
 
-* [Kubernetes Basics](/docs/tutorials/kubernetes-basics/) 여러분이 Kubernetes 시스템을 이해하고 기본 Kubernetes 기능을 시도할 때 도움을 주는 대화식 튜토리얼입니다.
+* [Kubernetes 기초](/docs/tutorials/kubernetes-basics/) 여러분이 Kubernetes 시스템을 이해하고 기본 Kubernetes 기능을 시도할 때 도움을 주는 대화식 튜토리얼입니다.
 
-* [Scalable Microservices with Kubernetes (Udacity)](https://www.udacity.com/course/scalable-microservices-with-kubernetes--ud615)
+* [Kubernetes를 사용한 확장성있는 마이크로 서비스 (Udacity)](https://www.udacity.com/course/scalable-microservices-with-kubernetes--ud615)
 
-* [Introduction to Kubernetes (edX)](https://www.edx.org/course/introduction-kubernetes-linuxfoundationx-lfs158x#)
+* [Kubernetes 소개 (edX)](https://www.edx.org/course/introduction-kubernetes-linuxfoundationx-lfs158x#)
 
-* [Hello Minikube](/docs/tutorials/hello-minikube/)
+* [안녕 Minikube](/docs/tutorials/hello-minikube/)
 
 ## 구성
 
-* [Configuring Redis Using a ConfigMap](/docs/tutorials/configuration/configure-redis-using-configmap/)
+* [ConfigMap을 활용한 레디스 구성하기](/docs/tutorials/configuration/configure-redis-using-configmap/)
 
 ## 상태없는 애플리케이션
 
-* [Exposing an External IP Address to Access an Application in a Cluster](/docs/tutorials/stateless-application/expose-external-ip-address/)
+* [클러스터의 애플리케이션에 접속할 수 있는 외부 IP 노출하기](/docs/tutorials/stateless-application/expose-external-ip-address/)
 
-* [Example: Deploying PHP Guestbook application with Redis](/docs/tutorials/stateless-application/guestbook/)
+* [Example: Deploying PHP Guestbook application with Redis예제: 레디스를 활용한 PHP 방명록 애플리케이션 배포하기](/docs/tutorials/stateless-application/guestbook/)
 
 ## 상태있는 애플리케이션
 
-* [StatefulSet Basics](/docs/tutorials/stateful-application/basic-stateful-set/)
+* [StatefulSet 기초](/docs/tutorials/stateful-application/basic-stateful-set/)
 
-* [Example: WordPress and MySQL with Persistent Volumes](/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/)
+* [예제: 영속 볼륨(Persistent Volumes)을 활용한 워드프레스와 MySQL](/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/)
```

Comment:
WordPress is also a product name but IMO "워드프레스" is fine since the translated word does not look awkward and the term is widely used to non-developers (e.g., marketing people) in Korea.